src/Bookings.js

```diff
@@ -1,18 +1,24 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
   const search = searchVal => {
     console.info("TO DO!", searchVal);
   };
 
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me`)
```

Comment:
Where you've done this logic for making the network request you could think about abstracting this code into a separate function called something like `fetchBookings` and then invoke it inside of `useEffect`